### PR TITLE
adds task 28, frontend AdminPostCreation component

### PR DESCRIPTION
**Pull request by alexm323**
First draft of admin post creation component (needs styling, but functionality is there)

### CODE REVIEW

**Review by indifferentghost — CHANGES_REQUESTED** [review]

File: packages/client/src/AdminPostCreation.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import { useState } from "react";
+
+// Data looks like this (for backend)
+// console.log({
+//     title:'string',
+//     body:'string',
+//     publish:boolean
```
[the remark this answers is not in the document]
Comment (relspeedwagon) [reply packages/client/src/AdminPostCreation.js]: Completed :)

File: packages/client/src/AdminPostCreation.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+// })
+
+let backend = "https://localhost3000";
```
[the remark this answers is not in the document]
Comment (relspeedwagon) [reply packages/client/src/AdminPostCreation.js]: Completed :)

File: packages/client/src/AdminPostCreation.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  const [title, setTitle] = useState("");
+  const [body, setBody] = useState("");
+  const [publish, setPublish] = useState(false);
+
+  const handleSubmit = async (e) => {
+    // we might want to go to the next page to see the published post (have to verify this)
+    e.preventDefault();
+    if (!title || !body) {
+      // lets remove this at some point before google deprecates it
+      alert("Please add a body/text");
```
[the remark this answers is not in the document]
Comment (relspeedwagon) [reply packages/client/src/AdminPostCreation.js]: All set :)

**Comment by relspeedwagon** [comment]
README.md should be fine, our add'n is REACT_APP_BACKEND for .env

We are confused as to why our branch appears to add anything to yarn.lock?  Need help reviewing please, TY!

**Comment by indifferentghost** [comment]
It doesn't look like there's any styles for this.

File: packages/client/src/AdminPostCreation.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
+import { useState } from "react";
+
+// Data looks like this (for backend)
+// console.log({
+//     title:'string',
+//     body:'string',
+//     isDraft:boolean
+// })
+let backend = process.env.REACT_APP_BACKEND;
+let endpoint = `${backend}/api/createPost`;
```
Comment (indifferentghost) [inline packages/client/src/AdminPostCreation.js:10]: These should be `const`

Comment (relspeedwagon) [reply packages/client/src/AdminPostCreation.js]: Resolved these, changed to "const" and committed here on GitHub :)

**Comment by relspeedwagon** [comment]
The remaining requested change should be taken care of through last commit--though I can no longer see the details of the request that still appears pending.  I'm sorry I may be missing something!

Other files show minor changes as result of linting.

**Review by indifferentghost — APPROVED** [review]

**Pull request merged**